sdk/communication/azure-mgmt-communication/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 1.0.0b2 (2020-09-22)
+
+**General breaking changes**
+
+Learn more about migrating to the next generation of the Azure Python SDK in the [migration guide for resource management](https://github.com/Azure/azure-sdk-for-python/blob/master/doc/sphinx/python_mgmt_migration_guide.rst)
+
+- Credential system has been completly revamped:
+
+  - `azure.common.credentials` or `msrestazure.azure_active_directory` instances are no longer supported, use the `azure-identity` classes instead: https://pypi.org/project/azure-identity/
```

Comment:
It seems you miss other breaking changes https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/resources/azure-mgmt-resource/CHANGELOG.md#1500b1-2020-06-17